platform/ui/src/components/studyList/CustomDateRangePicker.js

```diff
@@ -2,13 +2,12 @@
 //  https://github.com/airbnb/react-dates#initialize
 import 'react-dates/initialize';
 import 'react-dates/lib/css/_datepicker.css';
+import './CustomDateRangePicker.css';
```

Comment:
quick question: In some places, I see `.css` being used and in some other `.styl` as well as the use of CSS standards like BEM or just hyphenated lowercase words. Do we have any standards for this or is it something we can think about in the future? Just a curiosity to see how we can move forward in the future about this :)

Reply:
This is a great point! This is a conversation I want to have with the team at some point. Currently, everyone more or less employs their particular preference, or tries to adhere as closely to what they believe the current file's style is using.

Long-term, I think we'll either land on something like `tailwindcss`
See: https://tailwindcss.com/

Or CSS Modules, as we can scope classes to the component, and whatever naming convention the team prefers; but enforced with something like `stylelint`